### ISSUE TITLE
Add support for Windows 11

### ISSUE THREAD

**Issue by mmiretsky** [snakefoot/snaketail-net#160]
Please add support for Windows 11. Currently, the installer does not even start on Windows 11.

**Comment by snakefoot** [comment]
I'm guessing Windows 11 is protecting you from unsigned MSI-packages from untrusted source.

If you have ideas for to improve the WIX-project, then pull-request are most wellcome:

- https://github.com/snakefoot/snaketail-net/tree/master/SnakeTail.Wix

**Comment by evoisard** [comment]
Here, SnakeTail complains about missing .NET Framework 3.5 and Windows tries to download the pack. This is an issue on computers which are not connected to the internet. This is also a problem because .NET 3.5 is now obsolete and more an more users don't want this on their computer for security reasons...
What would be the effort to make SnakeTail use .NET 4 instead of 3.x? (afaik, it's more or less just a project parameter in Visual Studio...)
Anyway, many thanks for this very useful tool that we're using since years!

**Comment by snakefoot** [comment]
If you place the `Snaketail.exe.config`-file next to the SnakeTail.exe, then it should recognize that it supports all .NET runtimes.

**Comment by evoisard** [comment]
Indeed, I only put the executable plus an xml config file in my directory (no Snaketail.exe.config) ...

Then, from Windows Control Panel, I deactivated .NET 3.5 that I had just installed to make snaketail work. I tried to run snaketail to ensure the framework was not available anymore. It failed as expected. Then I added Snaketail.exe.config and this time it worked.

So this was the (simple) solution!

Thanks for the help and for the nice tool!
